src/OnboardingSPA/components/HeaderMenuPreview/stylesheet.scss

```diff
@@ -0,0 +1,112 @@
+/*COLOR VARIABLES*/
```

Comment:
Are we importing this stylesheet anywhere?? :)

Reply:
deleted the file ... the CSS class being used in the component is the same as the one for the livepreviewselectedable component